documentation/user/Information center.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: wiki
+title: Information center
+description: Downloaded ACE3 and have no idea where to start? This page serves as a document to help new players get started with things or get an answer to some of your questions.
+group: user
+order: 0
+parent: wiki
+---
+
+Downloaded ACE3 and have no idea where to start? This page serves as a document to help new players and mission makers understand what's available to them.
+
+
+- You don't know where to begin your ACE3 journey? [**Check out ACE3 features**](http://ace3mod.com/wiki/feature/)
+
+
+- You are a mission maker but you don't know what ACE3 have to offer? [**We have some documentation for you**](http://ace3mod.com/wiki/missionmaker/)
+
+
+- Are you searching for ACE3 classnames ? [**Here they are**](http://ace3mod.com/wiki/missionmaker/classnames.html)
+
+
+## 1. FAQ
+### 1.1 Features 
+<b>Q:</b> Where is X feature? </br>
+<b>A:</b> When it's done.</br>
+
+<b>Q:</b> Feature X was in ACE2/AGM/CSE where is it?</br>
+<b>A:</b> It's going to be ported at some point.</br>
+
+<b>Q:</b> Why was my feature request was closed on GitHub?</br>
+<b>A:</b> Because it's supposed to go [HERE](https://github.com/acemod/ACE3/issues/414/)
+
+<b>Q:</b> I want to disable (feature) how do I do it?</br>
+<b>A:</b> Simply delete the PBO.(note that some features depends on others, check dependencies before deleting anything).</br>
+
+### 1.2 Issues
+
+<b>Q:</b> Laser locking is broken, when are you going to fix it?</br>
+<b>A:</b> Fun fact, it isn't, you need to come from the direction of the laser, (laser is pointing to the east, you come from the west) and you drop the GBU, it will then guide itself to the target.</br>
+
+<b>Q:</b> I take vanilla damage with ACE 3.1.1</br>
+<b>A:</b> This has been fixed on ACE3 master and will be fixed in the next release, in the meantime Basic medical doesn't have that issue. </br>
+
+<b>Q:</b> I'm having dll errors.</br>
+<b>A:</b> Remove ACE3 from your Arma 3 folder and repeat the installation process (don't forget to re-download).</br>
+
+### 1.3 Compatibility
+
+<b>Q:</b>(mod) doesn't have some ACE3 features.</br>
+<b>A:</b>ACE3 isn't and can't be responsible for compatibility with every (mod), due it's size other (mod) authors are strongly encouraged to provide that from their side. Compatibility PBOs currently in ACE3 are there to kick-start and provide examples for (mod) authors.</br>
+
+<b>Q:</b> ACE3 cause issues in (mod).</br>
+<b>A:</b>ACE3 isn't and can't be responsible for all (mod) conflicts, due it's size other (mod) authors are strongly encouraged to provide that from their side.</br>
```

Comment:
Maybe how it's done on the website: `If you've found an issue with ACE3 please make sure that ACE3 is really the cause of the problem. To do this try to reproduce the issue with using only @cba_a3 and @ACE3 on a newly created mission.`